test/extended/images/mongodb_replica.go

```diff
@@ -0,0 +1,58 @@
+package images
+
+import (
+	"fmt"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/db"
+)
+
+var _ = g.Describe("[images][mongodb] openshift mongodb replication", func() {
+	defer g.GinkgoRecover()
+
+	const (
+		templatePath     = "https://raw.githubusercontent.com/openshift/mongodb/master/2.4/examples/replica/mongodb-clustered.json"
```

Comment:
depends on what it's testing, but in this case the intent should be to confirm the replication example we provide works, so it should indeed be using the template file from the external repository.

Now, i am doing some work to vendor external templates into origin.  This is not currently one of the templates being vendored, but if we do vendor it, then we could change this test to reference that.

But for now, i think the external reference is the right thing to do.